.github/workflows/manage-stale-issues-prs.yml

```diff
@@ -0,0 +1,44 @@
+name: 'Close stale issues and PRs'
+
+env:
+  STALE_AFTER_INACTIVE_DAYS: 90
+  CLOSE_AFTER_INACTIVE_DAYS: 7
+
+on:
+  schedule:
+    - cron: '1 3 * * 0'  # runs every Sunday at 03h01
+
+jobs:
+  stale:
+    permissions:
+      issues: write
+      pull-requests: write
+    if: github.repository_owner == 'geopython'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: 'actions/stale@v9'
+        with:
+          debug-only: true
+          enable-statistics: true
+          stale-issue-label: Stale
+          stale-pr-label: Stale
+          exempt-issue-labels: blocker
+          exempt-pr-labels: blocker
+          days-before-stale: ${{ env.STALE_AFTER_INACTIVE_DAYS }}
+          days-before-close: ${{ env.CLOSE_AFTER_INACTIVE_DAYS }}
+          remove-stale-when-updated: true
+          stale-issue-message: >
```

Comment:
```yaml
          stale-issue-message: >
            As per [RFC4](https://pygeoapi.io/development/rfc/4), this Issue has been inactive for ${{env.STALE_AFTER_INACTIVE_DAYS }} days. In order to manage maintenance burden, it will be automatically closed in ${{ env.CLOSE_AFTER_INACTIVE_DAYS }} days.
          stale-pr-message: >
            As per [RFC4](https://pygeoapi.io/development/rfc/4), this Pull Request has been inactive for ${{env.STALE_AFTER_INACTIVE_DAYS }} days. In order to manage maintenance burden, it will be automatically closed in ${{ env.CLOSE_AFTER_INACTIVE_DAYS }} days.
          close-issue-message: >
            As per [RFC4](https://pygeoapi.io/development/rfc/4), this Issue has been closed due to there being no activity for more than ${{ env.STALE_AFTER_INACTIVE_DAYS }} days.
          close-pr-message: >
            As per [RFC4](https://pygeoapi.io/development/rfc/4), this Pull Request has been closed due to there being no activity for more than ${{ env.STALE_AFTER_INACTIVE_DAYS }} days.
```